rir/src/api.cpp

```diff
@@ -366,3 +366,5 @@ bool startup() {
 }
 
 bool startup_ok = startup();
+
```

Comment:
I do not get it. Why do we send to GNU-R and empty function for the cleanup?

Reply:
Was for a #474, although I can just use `on.exit`